JASP-Desktop/analysis/analysisform.cpp

```diff
@@ -798,7 +798,9 @@ bool AnalysisForm::runWhenThisOptionIsChanged(Option *option)
 	BoundQMLItem* control = getBoundItem(option);
 	JASPControlBase* item = control ? control->item() : nullptr;
 
-	emit optionChanged(item);
+	emit valueChanged(item);
```

Comment:
Why would you want to send `valueChanged(nullptr)`, does that make sense?

Reply:
The Form valueChanged signal does not need per se to know which item is changed (in the code I gave you for the last PR for example). So even if the item is not found (it could be the case with controls made dynamically (like rowControls), the signal should be emit.

Reply:
Ok!